src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -115,24 +111,24 @@ These procedures to configure your Android app with Gradle and Android Studio al
 
 3. Set app permissions: Ensure that your Android app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml` file:
 
-   ```
-   <uses-permission android:name="android.permission.INTERNET" />
-   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-   ```
+  ```
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  ```
 4. To start New Relic's mobile monitoring agent: In your **Default Activity** (as defined in your **Manifest**), import the `NewRelic` class:
 
-   ```
-   import com.newrelic.agent.android.NewRelic;
-   ```
+  ```
+  import com.newrelic.agent.android.NewRelic;
+  ```
 
-   <Callout variant="important">
-     We do not support starting the mobile monitoring agent in other classes, as that can cause unexpected or unstable behavior.
-   </Callout>
+  <Callout variant="important">
+    We do not support starting the mobile monitoring agent in other classes, as that can cause unexpected or unstable behavior.
+  </Callout>
 5. To initialize mobile monitoring: In the `onCreate()` method, add this call:
 
-   ```
-   NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
-   ```
+  ```
+  NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
+  ```
```

Comment:
Let's add Java syntax highlighting to all these code samples (on the other install/configure Android pages too).
```suggestion
  ```java
  NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
  ```
```